repositories/src/main/scala/cromwell/pipeline/datastorage/dao/repository/ProjectRepository.scala

```diff
@@ -6,25 +6,40 @@ import cromwell.pipeline.datastorage.dto.{ Project, ProjectId }
 
 import scala.concurrent.Future
 
-class ProjectRepository(pipelineDatabaseEngine: PipelineDatabaseEngine, projectEntry: ProjectEntry) {
+trait ProjectRepository {
+
+  def getProjectById(projectId: ProjectId): Future[Option[Project]]
+  def getProjectsByName(name: String): Future[Seq[Project]]
+  def addProject(project: Project): Future[ProjectId]
+  def deactivateProjectById(projectId: ProjectId): Future[Int]
+  def updateProjectName(updatedProject: Project): Future[Int]
+  def updateProjectVersion(updatedProject: Project): Future[Int]
```

Comment:
[minor] Please be consistent with your code style. Methods in other traits have line separators between them, but not in this trait. I.e. change
```
def foo = ???
def bar = ???
def buzz = ???
```
to
```
def foo = ???

def bar = ???

def buzz = ???
```

Reply:
@myazinn done